modules/appeals_api/app/workers/concerns/appeals_api/character_utilities.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module CharacterUtilities
+    extend ActiveSupport::Concern
+
+    included do
+      def transliterate_for_centralmail(str)
+        I18n.transliterate(str.to_s).gsub(/[^a-zA-Z\-\s]/, '').strip.first(50)
```

Comment:
Note this may not have ALL the characters centralmail allows, but it seemed a decent set to get us through the bug.